### PR TITLE
Run forest calibnet node

### PR DESCRIPTION
**Pull request by samuelarogbonlo**
**Summary of changes**
Changes introduced in this pull request:
- DO set up to tun forest calibnet 
- Terraform and ansible to manage infrastructure setup 
- Secrets managed by user 

**Other information and links**

### CODE REVIEW

File: forest-node/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+To implement the infrastructure, run the following:
+- Create `ssh-key` to be added to DigitalOcean list and store the fingerprint for use in the next step; you can check more details [here](https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/to-team/)
+- Create a space on DigitalOcean with any preferred unique name and add the bucket name in the `backend.tf` file. 
+- Generate `digitalocean_api_token` from DigitalOcean console; you can check [here](https://docs.digitalocean.com/reference/api/create-personal-access-token/) for more details.
+- Populate the `terraform.tfvars` file with the values of the following
+    - `new_key_ssh_key_fingerprint`
+    - `firewall_name`
+    - `name`
+- Set all necessary environment variables to the terminal permanently by adding the to a shell profile. To do this, run `nano ~/.bash_profile` then add the following to the end of the file
```
Comment (lemmih) [inline forest-node/README.md:26]: I think we can just leave this out. Saying that the three environment variables need to be defined is good enough and then we don't have to mention `nano` or `.bash_profile`.

Comment (samuelarogbonlo) [reply forest-node/README.md]: Noted

File: forest-node/terraform/Makefile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+destroy:
+	@terraform destroy -var-file="terraform.tfvars"
+<<<<<<< HEAD
+	
+	
+	
+=======
+	
+>>>>>>> 3155d71 (sorting trailing new lines)
```
Comment (lemmih) [inline forest-node/terraform/Makefile:18]: Merge mishap.

File: forest-node/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+To implement the infrastructure, run the following:
+- Create `ssh-key` to be added to DigitalOcean list and store the fingerprint for use in the next step; you can check more details [here](https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/to-team/)
+- Create a space on DigitalOcean with any preferred unique name and add the bucket name in the `backend.tf` file.
+- Generate `digitalocean_api_token` from DigitalOcean console; you can check [here](https://docs.digitalocean.com/reference/api/create-personal-access-token/) for more details.
+- Populate the `terraform.tfvars` file with the values of the following
+    - `new_key_ssh_key_fingerprint`
+    - `firewall_name`
+    - `name`
+- Set all necessary environment variables to the terminal permanently by adding the to a shell profile.
```
Comment (lemmih) [inline forest-node/README.md:26]:
```suggestion
- Set all necessary environment variables to the terminal permanently by adding them to a shell profile.
```

Comment (samuelarogbonlo) [reply forest-node/README.md]: Noted.

**Review by lemmih — APPROVED** [review]
Alright.

**Pull request merged**